src/audio/dai.c

```diff
@@ -10,6 +10,7 @@
 #include <sof/audio/pipeline.h>
 #include <sof/common.h>
```

Comment:
Title should contain a verb. Lets go with: drivers: imx: Add ESAI support

Also, please add some pointers to the documentation. Just a give a link to i.MX8QXP RM with the chapter for ESAI description.

Reply:
Mostly understood actually, had a statement without effect because it was in fact a repeated statement of setting the exact same bits. Will push with updated comments.

Reply:
Updated commit description (probably not yet the final one though).